src/re_riak.erl

```diff
@@ -758,12 +758,24 @@ nodes(Cluster) ->
     end.
 
 node_config(_, Node) ->
-    load_patch(Node),
-    case remote(Node, re_riak_patch, effective_config, []) of
-        {error, legacy_config} ->
-            [{error, not_found, [{error, <<"Legacy configuration files found, effective config not available.">>}]}];
-        Config ->
-            [{config, Config}]
+    case load_patch(Node) of
+        ok ->
+            try
+                case remote(Node, re_riak_patch, effective_config, []) of
+                    {error, legacy_config} ->
+                        [{error, not_found, [{error, <<"Legacy configuration files found, effective config not available.">>}]}];
+                    Config ->
+                        [{config, Config}]
+                end
+            catch
+                Exception:Reason ->
+                    Error = list_to_binary(io_lib:format("~p:~p", [Exception,Reason])),
+                    lager:info("~p:~p", [Exception, Reason]),
+                    lager:info("Backtrace: ~p", [erlang:get_stacktrace()]),
+                    [{config, [{error, Error}]}]
```

Comment:
@drewkerrigan I duplicated code here from `handle_error/1`, so I probably want to go back and refactor it slightly.  I couldn't re-use it because `handle_error/1` always returned `config` as the first key of the proplist.